main.rb

```diff
@@ -0,0 +1,36 @@
+require_relative 'app'
+
+def main
+  app = App.new
+  app.run
+end
+
+def display_options
+  puts '1. List all books'
+  puts '2. List all persons'
+  puts '3. Create a person'
+  puts '4. Create a book'
+  puts '5. Create a rental'
+  puts '6. List all rentals for a given person id'
+  puts '7. Quit'
```

Comment:
- [ ] `Optional`
- [ ] For better UX  Have a way to quit way with something like "Thank you for using this app!"